packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -126,20 +126,24 @@ export default {
          * @returns {boolean}
          */
         hasRatingAvailable () {
-            return this.ratingDisplayType !== 'noRating';
+            return this.reviewCount > 0;
         }
     },
 
     methods: {
         /**
          * Gets the correct rating display format from translations.
          *
-         * @todo - If the component is using `short` as a `starRatingSize` we shouldn't display text
-         * alongside it for now. (TBC with design - ticket in backlog).
+         * 1. If reviewCount is Zero, return "No ratings yet".
+         * 2. If reviewCount is available, return `ratingDisplayType` .
          *
-         * @returns {string}
+         * @returns {String}
```

Comment:
Is the case change necessary? Seems to break consistency from above

Reply:
My bad.